lib/survey_gizmo/configuration.rb

```diff
@@ -1,33 +1,36 @@
 require 'survey_gizmo/faraday_middleware/parse_survey_gizmo'
 
 module SurveyGizmo
-  class << self
-    attr_writer :configuration
```

Comment:
This is breaking backward compatibility. Though I don't think a lot of integrations were building their own `SurveyGizmo::Configuration` instance to set it with `SurveyGizmo.configuration = config`.

Reply:
Can't we just add this method back? setting the thread variable this time?

Reply:
You're completely right. I didn't think about it enough and thought that it was going to be confusing because it wouldn't change the last known configuration. But we can actually save it. That's what I did in 602d608 and I updated the README.

In d8baa49 I removed the spec which was failing about shared references seems we agreed on it being ok.